### ISSUE TITLE
Problem in first applications - 

### ISSUE THREAD

**Issue by MiguelRomeiro** [pymanga/pyMANGA#345]
Hello, 

When running the example setup "OGS3D_SAZOI_BETTINA.xml", things start running smoothly, but then come to a halt when dealing with OGS. The code reports the error (Ogs calculation failed). It comes from ".\ResourceLib\BelowGround\Individual\OGS\OGS.py" at line 428, in "runOGSandWriteFiles". 

As far as I understand, this piece of code calls another command prompt to run OGS (from ".\ResourceLib\BelowGround\OGS\bin\ogs. exe"). Do you have any ideas on why this might be happening, and how to solve it?

I am currently using pyMANGA for my Master's thesis, but have been strugling with this part (been stuck on it for a few days). Any help would be very much appreciated!

**Comment by mcwimm** [comment]
Hi, sorry for the late reply. Can you give us some background information, e.g., which OGS version and system you are using?

**Comment by MiguelRomeiro** [comment]
Don't worry about it, I'm glad you answered! And my bad: the OGS version
I'm using is 6.4.0. I'm trying to run it in Windows 8 (maybe that's the
problem?)

Miguel Piovesana Pereira-Romeiro
Instituto de Biociências - Universidade de São Paulo (IB / USP)
Laboratory of Theory in ecology and evolution
<https://www.ifi.unicamp.br/~flaviam/>


Em ter., 25 de jun. de 2024 às 12:14, mcwimm ***@***.***>
escreveu:

**Comment by mcwimm** [comment]
Okay. Thanks for the update. OGS version 6.4.0 should work, but we never tested it using Windows 8. Lets see if we can narrow down the issue a bit.
Can you please check what happens when you test the following
1) Start OGS from the command line by running the following from the OGS directory
``
.\bin\ogs.exe -h
``
This should list the possible flags.
2) If this works, try to start OGS with an OGS project file by running the following from the pyMANGA root directory
```
...\pyMANGA> .\ResourceLib\BelowGround\Individual\OGS\bin\ogs.exe .\Benchmarks\ExampleSetups\OGSExampleSetup\OGS\ogs_pr
oject.prj
```
If OGS is running, you should see some infos and warnings....

**Comment by MiguelRomeiro** [comment]
Hi, I think I figured out the issue. 

In the First Applications documentation, there is a step that says to copy the _bin_ folder (from the original OGS zip file download) into the following path : "pyMANGA-master\ResourceLib\BelowGround\OGS". Then, figure 4 shows a different path ("pyMANGA-master\ResourceLib\BelowGround\OGS" - which I assume to be a previous version of pyMANGA). 

It seems that neither paths exist in the current version. Originally, I had manually created a "OGS" folder within "BelowGround" (as the path indicated). But then, after seeing your message, I suspected that my _bin_ folder was in the wrong place. After I changed it to ".\ResourceLib\BelowGround\Individual\OGS\", everything went smoothly! I'd recommend updating the documentation with the current, correct paths and an up to date figure in the place of figure 4.

As for warnings, the only one that showed up indicated deprecated parts of the code, but I wasn't able to identify where.
